src/app/blog/components/post-detail/post-detail.component.html

```diff
@@ -23,8 +23,9 @@ <h1 data-aos="fade-in-up" data-aos-offset="100" class="text-primary">BLOG</h1>
   -->
 
   <p-fieldset *ngIf="selectedTopic">
+    <h4>Laisser un commentaire<br>Votre adresse de messagerie ne sera pas publiee. Les Champs Obligatoires sont indicquees avec *</h4><hr>
     <p *ngFor="let c of selectedTopic.comments">
-      <span class="list-group-irem"> {{c.msg}} </span>:{{c.createdDate}}
+      <i style="color: blue">{{c.author}}</i> : <span class="list-group-irem"> {{c.msg}} </span><i style="float: right;">{{c.createdDate | date: 'short'}}</i><!-- @Ghislain please can you help with how to display dynamic time in angular-->
```

Comment:
style="float: right;  for what? why do you use your own ??   We got 2 Frameworks for that

Reply:
yes but i did see a way to display the time at right angle there